website/content/docs/enterprise/consistency.mdx

```diff
@@ -0,0 +1,212 @@
+outside the scope of this document, but suffice it to say that both performance
+replication and performance standbys use something called log shipping to stay
+in sync with the upstream source of writes.  The mitigation used by nodes
+sending writes upstream via RPC is to look at the WAL index in the response
+and wait up to 2 seconds to see the that WAL index appear in the logs being
+shipped from upstream.  If the WAL index is seen, the Vault node handling the
+request that resulted in RPCs can return: it knows that any subsequent reads
+will be able to see the value that was just written.  If the WAL index isn't
+seen within those 2 seconds, the Vault node completes the request and hopes
+for the best.
+
+This mitigation option still exists in Vault 1.7, though now there is a
+configuration option to adjust the wait time.
+
+(TODO: reference new config option)
+
+## Vault 1.7 Mitigations
+
+There are now a variety of other mitigations available:
+* per-request option to blindly forward the request to the active node
+* per-request option to conditionally forward the request to the active node
+  if it would otherwise result in a stale read
+* per-request option to fail requests if they might result in a stale read
+* Vault Agent configuration to do the above for proxied requests
+
+The remainder of this document describes the tradeoffs of these mitigations and
+how to use them.
+
+### Blind Forwarding (Performance standbys only)
+
+The simplest solution to never experience stale reads from a performance standby
+is to provide the following HTTP header in the request:
+
+```
+X-Vault-Forward: active-node
+```
+
+The drawback here is that if all your requests are forwarded to the active node,
+you might as well not be using performance standbys.  So this mitigation only
+makes sense to use selectively.
+
+This mitigation will not help with stale reads relating to performance replication.
+
+### Conditional Forwarding (Performance standbys only)
+
+As of Vault Enterprise 1.7, all requests that modify storage now return a new
+HTTP response header:
+
+```
+X-Vault-Index: <base64 value>
+```
+
+To ensure that the state resulting from that write request is visible to a
+subsequent request, add these headers to that second request:
+
+```
+X-Vault-Index: <base64 value taken from previous response>
+X-Vault-Inconsistent: forward-active-node
+```
+
+The effect will be that the node handling the request will look at the state
+it has locally, and if it doesn't contain the state described by the X-Vault-Index
+header, the node will forward the request to the active node.
+
+The drawback here is that when requests are forwarded to the active node,
+performance standbys provide less value.  If this happens often enough
+the active node can become a bottleneck, limiting the horizontal read scalability
+performance standbys are intended to provide.
+
+### Retry stale requests
+
+As of Vault Enterprise 1.7, all requests that modify storage now return a new
```

Comment:
I like that you've repeated this here, in case someone is directed to just this one section.